pages/osx/defaults.md

```diff
@@ -0,0 +1,23 @@
+# defaults
+
+> Access OS X user defaults
+
+- read system default value
+
+`defaults read {{domain}} {{key}}`
+
+- read default values of applications
+
+`defaults read -app {{app_name}} {{key}}`
+
+- write key value
+
+`defaults write {{domain}} {{key}} {{-type}} {{value}}`
+
+- Speed up Mission Control animations
```

Comment:
@waldyrious do we still need to remove it. i feel this example might be ok. in this case, maybe we just can merge it.